jellyfin-api/src/commonMain/kotlin/org/jellyfin/sdk/api/client/HttpClientOptions.kt

```diff
@@ -14,17 +17,17 @@ public data class HttpClientOptions(
 	 * Timeout for a complete HTTP request in milliseconds.
 	 * Defaults to 30 seconds.
 	 */
-	val requestTimeout: Long = 30_000,
+	val requestTimeout: Duration = 30.seconds,
```

Comment:
Comments for this and the other timeout variables need to be updated.

Reply:
Done!